tests/functional/profiles/test_profile_dir.py

```diff
@@ -84,18 +88,8 @@ def environ(env):
                 os.environ[key] = value
 
 
-# Use this if you need to capture the standard out in a test
-def run_dbt_and_capture_stdout(args: List[str] = None, expect_pass=True):
-    stringbuf = io.StringIO()
-    with redirect_stdout(stringbuf):
-        res = run_dbt(args, expect_pass=expect_pass)
-    stdout = stringbuf.getvalue()
-
-    return res, stdout
-
-
 class TestProfiles:
-    def dbt_debug(self, project_dir_cli_arg=None, profiles_dir_cli_arg=None):
+    def dbt_debug(self, project, project_dir_cli_arg=None, profiles_dir_cli_arg=None):
```

Comment:
Excited about all the simplification you were able to do for the `tests/functional/profiles/test_profile_dir.py` test!

Is adding `project` essential? If not, what do you think about leaving it as the following?
```suggestion
    def dbt_debug(self, project_dir_cli_arg=None, profiles_dir_cli_arg=None):
```

Reply:
@dbeatty10 It is because you need the profile for the run command to work with run_dbt or run_dbt_and_capture

Reply:
Yikes, I was wrong. This _was_ needed at some point, but now I can go in and remove it. Thanks for the callout 🙇‍♀️ 😅 I need to dive in the next time I find the error I described and I identify it's a context-specific bug or something. I've run into it too many times for it to be fake, but also, my understanding of it has now been shifting as a result of this convo.

tl;dr merged your change

Reply:
No worries!

Will be a very happy day once this is merged and my custom `run_dbt_and_capture_stdout` is no longer a thing!

Will re-review shortly.